terraform/modules/agreements-db/provider.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_providers {
+    cloudfoundry = {
+      source  = "cloudfoundry-community/cloudfoundry"
+      version = "0.13.0"
+    }
+    aws = {
+      version = ">= 2.7.0"
+
+    }
+  }
+}
+
+provider "cloudfoundry" {
```

Comment:
Could this provider configuration be declared once in the root module and inherited by the child modules (I know the provider _requirements_ block has to be explicitly declared in each)?

Reply:
Good call - that seems to work. I think I can also remove the aws provider requirements too